app/battle_preparations/apply_armour.py

```diff
@@ -0,0 +1,9 @@
+class Armour:
+    @staticmethod
+    def armour_protection(armors: list) -> int:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]